content/en/docs/Writing policies/validate.md

```diff
@@ -1203,9 +1203,7 @@ Regardless of where the disallowed `type: Unconfined` field is specified, Kyvern
 
 In situations where a more nuanced approach to pod security is required, Kyverno introduces the `exclude.restrictedFields` and `exclude.values[]` object. They allow users to exempt specific controls within a profile while applying others, providing flexibility to tailor security configurations based on the specific requirements of each pod or container.
 
-To leverage fine-grained exemptions effectively, users can utilize the `podSecurity.exclude[]` object in their pod configuration YAML. This allows for the exclusion of specific controls, providing a customized security approach without compromising overall compliance with Pod Security Standards.
-
-For example, the Capabilities control in the baseline profile mandates that the `securityContext.capabilities[]` field be not set to anything other than the defined values; `FOO` and `BAR` not being among those. The below policy enforces the baseline profile across the entire cluster while exempting the Capabilities control from all nginx images that have values `FOO` and/or `BAR`.
+For example, the baseline profile includes a control called Capabilities, which requires the `securityContext.capabilities[]` field to only have certain values. However, it doesn't allow `FOO` and `BAR` as valid values. The following policy enforces the baseline profile for the entire cluster but exempts the Capabilities control for all nginx images that have values `FOO` and/or `BAR`.
```

Comment:
I'm not sure why you deleted the paragraph in line 1206. I understand you want to replace 1208 with your suggestion, but I still see 1206 as useful.

Reply:
@MariamFahmy98 ⬆️